server/src/main/java/com/objectcomputing/checkins/services/team/TeamServicesImpl.java

```diff
@@ -63,4 +82,23 @@ public Team update(Team team) {
         }
         return teams;
     }
+
+    // public void delete(@NotNull UUID id) {
+    //     teamsRepo.deleteById(id);
+    // }
+
```

Comment:
Might want to remove old, commented out method =)

Reply:
updated, thanks =)